app/models/provider_user.rb

```diff
@@ -48,6 +48,10 @@ def can_manage_users?
     provider_permissions.exists?(manage_users: true)
   end
 
+  def can_manage_organisations?
+    ProviderInterface::ProviderRelationshipPermissions.exists?(training_provider: providers)
```

Comment:
See previous comment, this is a little dangerous.

Reply:
How about adding a `can_manage_organisations?(provider:)` instead within `ProviderAuthorisation`, along the lines of `can_view_safeguarding_information?(course:)`?

Some of these methods within `ProviderUser` make sense, especially for quick global checks, such as `can_manage_users?`. If we need we a quick UI-related decisions (e.g. whether to show the Users link in the provider interface header) they are fine, but it's too easy to start relying on these in other parts of the code when we should be more specific (is this user allowed to manage users for this specific provider...).

Others should probably be removed, such as `can_view_safeguarding_information_for?(provider)`, which is no longer used, from what I can see.

Reply:
:+1: I think this PR is a little behind how we intend to manage this permission, I've since added an explicit user-level permission for this https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/blob/master/db/schema.rb#L317 so perhaps we should use this on the ProviderUser.
I agree we should be checking the orgs carefully, I've done something similar here https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/2202/commits/2b08541e4931d55854a73d6aba363dd48d1f000f#diff-b4b81af4516e63df181af81c4c2c7d37R23 in a before filter for organisations pages. Will update.

Reply:
@mnacos https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/2231/commits/26ad3e533832eb611ef31ecf73a22de4e9e6cd50 removes the `can_view_safeguarding_information_for?(provider)` method. Thanks for spotting this.
I've now started using the user-level permission `can_manage_organisations` in conjunction with a check on the training provider: https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/blob/64852d66c3378a87206797339490eb6617697632/app/controllers/provider_interface/provider_relationship_permissions_controller.rb#L71-L74